Description: Patch out Canonical support information used for Ubuntu - We're Debian
 While in theory this could be managed with dpkg-vendor and some IFDEFery,
 it's just easier to patch it out.  This code hasn't changed in years, so the
 patch shouldn't take any effort to maintain.
Author: Scott Kitterman <scott@kitterman.com>
Origin: vendor
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=791875
Forwarded: not-needed
Last-Update: 2016-07-14

Index: muon-5.6.0/src/DetailsTabs/MainTab.cpp
===================================================================
--- muon-5.6.0.orig/src/DetailsTabs/MainTab.cpp	2016-07-14 21:16:59.311134381 -0400
+++ muon-5.6.0/src/DetailsTabs/MainTab.cpp	2016-07-14 21:16:59.303134381 -0400
@@ -178,19 +178,19 @@
     m_packageShortDescLabel->setText(m_package->shortDescription());
 
     m_descriptionBrowser->setText(m_package->longDescription());
-
+    // Patch out Ubuntu only support information
     // Append a newline to give a bit of separation for the support string
-    m_descriptionBrowser->append(QString());
-    if (m_package->isSupported()) {
-        QDateTime endDate = m_package->supportedUntil();
-        m_descriptionBrowser->append(i18nc("@info Tells how long Canonical, Ltd. will support a package",
-                                        "Canonical provides critical updates for %1 until %2.",
-                                        m_package->name(), endDate.date().toString()));
-    } else {
-        m_descriptionBrowser->append(i18nc("@info Tells how long Canonical, Ltd. will support a package",
-                                        "Canonical does not provide updates for %1. Some updates "
-                                        "may be provided by the Ubuntu community.", m_package->name()));
-    }
+    //m_descriptionBrowser->append(QString());
+    //if (m_package->isSupported()) {
+    //    QDateTime endDate = m_package->supportedUntil();
+    //    m_descriptionBrowser->append(i18nc("@info Tells how long Canonical, Ltd. will support a package",
+    //                                    "Canonical provides critical updates for %1 until %2.",
+    //                                    m_package->name(), endDate.date().toString()));
+    //} else {
+    //    m_descriptionBrowser->append(i18nc("@info Tells how long Canonical, Ltd. will support a package",
+    //                                    "Canonical does not provide updates for %1. Some updates "
+    //                                    "may be provided by the Ubuntu community.", m_package->name()));
+    //}
 }
 
 void MainTab::emitSetInstall()
